sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -75,8 +75,9 @@ public final class BridgeInternal {
     private BridgeInternal() {}
 
     @Warning(value = INTERNAL_USE_ONLY_WARNING)
-    public static CosmosDiagnostics createCosmosDiagnostics(DiagnosticsClientContext diagnosticsClientContext) {
-        return new CosmosDiagnostics(diagnosticsClientContext);
+    public static CosmosDiagnostics createCosmosDiagnostics(
```

Comment:
I see you already have an implementation bridge helper method for this, can we use that here and not break this if we don't have to?

Reply:
removed the method in BridgeINternal